Source/MgsLib/LibDG.cpp

```diff
@@ -645,12 +645,56 @@ MGS_FUNC_IMPLEX(0x401A4F, LibGvInitDispEnv_401A4F, LIBDG_IMPL);
 const char k0132_byte_650174[4] = { 0, 1, 3, 2 };
 
 void CC GV_kmd_zmd_file_handler_helper_4028C1(kmdObject* pKmdObj, kmdObject* pParentObj);
-MGS_FUNC_IMPLEX(0x4028C1, GV_kmd_zmd_file_handler_helper_4028C1, false); // TODO
+MGS_FUNC_IMPLEX(0x4028C1, GV_kmd_zmd_file_handler_helper_4028C1, LIBDG_IMPL); // TODO
 
 
 void CC GV_kmd_zmd_file_handler_helper_4028C1(kmdObject* pKmdObj, kmdObject* pParentObj)
 {
-   
+    unsigned int indexFlags = 0;
+    BYTE* vertexIndices = pKmdObj->indexOfs_3C;
+    for (int i = pKmdObj->field_4_numFaces * 4; i > 0; --i, ++vertexIndices)

```

Comment:
Even though the actual loops are backwards in the game I think we should use regular loops as its easier to understand.